nlp/2021_07_07_neural_keyword_extraction/neural_keyword_extraction.ipynb

```diff
@@ -0,0 +1,1520 @@
+{
+  "nbformat": 4,
+  "nbformat_minor": 0,
+  "metadata": {
+    "colab": {
+      "name": "Neural Keyword Extraction Tip-of-the-week.ipynb",
+      "provenance": [],
+      "collapsed_sections": [
+        "Ak0IG15fsUVV"
+      ],
+      "toc_visible": true
+    },
+    "kernelspec": {
+      "name": "python3",
+      "display_name": "Python 3"
+    }
+  },
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ne2ic1nmhs0n"
+      },
+      "source": [
+        "# Neural Keyword Extraction"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "j78lHqRwsUeU"
+      },
+      "source": [
+        "In a [pervious quick tip](https://github.com/ml6team/quick-tips/tree/main/nlp/2021_03_18_pke_keyword_extraction) we looked at [`pke`](https://boudinfl.github.io/pke/build/html/index.html) as a replacement for Gensim's [recently removed keyword extraction module](https://github.com/RaRe-Technologies/gensim/releases/tag/4.0.0).\n",
```

Comment:
For readability, I would chop up the first section in a few subsections and add a little bit more bullets. Hopefully the following image makes this a bit clear:
![image](https://user-images.githubusercontent.com/22496343/124916776-56214500-dff3-11eb-83ad-702dff438838.png)